units/initrd-parse-etc.service.in

```diff
@@ -21,7 +21,10 @@ OnFailureJobMode=replace-irreversibly
 [Service]
 Type=oneshot
 
-ExecStart={{ROOTLIBEXECDIR}}/systemd-sysroot-fstab-check
+# FIXME: once dracut is patched to install the symlink, change to:
+# ExecStart={{ROOTLIBEXECDIR}}/systemd-sysroot-fstab-check
+Environment=SYSTEMD_INVOKED_AS=systemd-sysroot-fstab-check
+ExecStart={{SYSTEM_GENERATOR_DIR}}/systemd-fstab-generator
```

Comment:
you are aware of the `@` syntax ExecStart= supports?

Reply:
So yeah, for invocations in a unit this would work. But not for invocations on the command-line, which I think is useful too. But I don't have a strong view on this… Preferences?

Reply:
I changed the unit to use `@`.